pom.xml

```diff
@@ -18,6 +18,10 @@
         <testng.version>7.4.0</testng.version>
         <assertj-core.version>3.20.2</assertj-core.version>
         <calculator-app.version>1.0</calculator-app.version>
+        <webdrivermanager.version>4.4.3</webdrivermanager.version>
+        <selenium-java.version>3.141.59</selenium-java.version>
+        <slf4j-simple.version>1.7.31</slf4j-simple.version>
+        <slf4j-api.version>1.7.31</slf4j-api.version>
```

Comment:
JFYI, these two properties could be combined into one

Reply:
I combined these properties into one